src/test/java/org/apache/commons/lang3/BooleanUtilsTest.java

```diff
@@ -321,7 +321,7 @@ public void test_toBoolean_String() {
         assertFalse(BooleanUtils.toBoolean("false"));
         assertFalse(BooleanUtils.toBoolean("a"));
         assertTrue(BooleanUtils.toBoolean("true")); // interned handled differently
-        assertTrue(BooleanUtils.toBoolean(new StringBuffer("tr").append("ue").toString()));
+        assertTrue(BooleanUtils.toBoolean("tr" + "ue"));
```

Comment:
Yes I understand. But the old one didn't make sense as well :-) I think we can drop this assertion completely.